### DISCUSSION TITLE
Ленивцы против единорогов.

### DISCUSSION THREAD

**Discussion by dzmitry-varabei** [rolling-scopes/rsschool-app#912] [Announcements]
### Вести с фронта. 
1) Сегодня 30 сентября в 9:25 один из ленивцев был замечен на странице, которая не существует.
![telegram-cloud-photo-size-2-5379734580677948738-y](https://user-images.githubusercontent.com/2429680/135437028-3d3dd865-a19f-4e35-b80a-7f5206ecdb52.jpg)
2) Несколько отлично подготовленных ленивцев смогли закрепиться на незначительном количестве пользовательских профилей.
![image](https://user-images.githubusercontent.com/2429680/135437686-5e7fcec7-78d9-4a99-b999-23a8fc59ed1a.png)

**Comment by AlreadyBored** [comment]
Кажется, я заметил, что ещё один из них промелькнул на другой странице...
Продолжаю наблюдение.

**Comment by valerydluski** [comment]
https://app.rs.school/sloth/caught
пойман)

**Comment by NMakarevich** [comment]
https://app.rs.school/heroes
Нашел стаю ленивцев)

**Reply by igorlyatskiy** [reply]
![image](https://user-images.githubusercontent.com/77933579/138151817-482c3421-5260-4b9b-8e80-1cc308c32df0.png)

**Comment by natein** [comment]
У https://app.rs.school/profile?githubId=jetalai3 обнаружен ленивец в бейджах
